### DISCUSSION TITLE
RNode LoRa Interface could not be created

### DISCUSSION THREAD

**Discussion by aj5r1gg** [markqvist/Reticulum#412] [Help & Questions] [answered]
I have an error showing up as:

`[Error] The interface "RNode LoRa Interface" could not be created. Check your configuration files for errors!`
`[Error] The contained exception was: '<' not supported between instances of 'NoneType' and 'int'`

My Reticulum config file is:
`[[RNode LoRa Interface]]`
`type = RNodeInterface`
`interface_enabled = True`
`port = /dev/ttyUSDB0`
`frequency = 867200000`
`bandwidth = 125000`
`txpower = 7`
`spreadfactor = 8`
`codingrate = 5`

There is no less than sign "<" anywhere that I can find.

Please advise how to correct this.

**Comment by faragher** [comment, marked as answer]
Short answer: `spreadfactor` should be `spreadingfactor`

Wider answer:

When you see something like `'<' not supported between instances of 'NoneType' and 'int'` it's because there's a test somewhere that's getting an uninitialized value. In this case it's checking to see if your SF is less than a number. Since "spreadingfactor" is uninitialized, it's None. 

Did you copy this configuration file or type it yourself? If documentation is incorrect, it needs to be fixed.

**Reply by aj5r1gg** [reply]
My apologies @faragher... It was my poor copying... Thank you for your input.
